setup.cfg

```diff
@@ -29,6 +29,8 @@ install_requires =
     elasticsearch==8.2.3
     fastapi==0.78.0
     gunicorn==20.1.0
+    h11==0.12.0
```

Comment:
Not sure this is used!

Reply:
I'm okay to temporarily pin it but add a comment to explain why we've pinned this secondary dependency. We will have to deal with Renovate every week until the latest `h11` release will be compatible with `httpcore`...

Reply:
Ok then.